src/components/merge_results/MergeResultsWizardContributorsTab.tsx

```diff
@@ -1,18 +1,54 @@
+import { Divider, FormControlLabel, Radio, RadioGroup } from '@mui/material';
 import { useContext } from 'react';
 import { useFormContext, useWatch } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { Contributor } from '../../types/contributor.types';
 import { Registration } from '../../types/registration.types';
+import { isOnImportPage } from '../../utils/urlPaths';
 import { MergeResultsWizardContext } from './MergeResultsWizardContext';
 import { CompareContributor } from './fields/CompareContributor';
 
+enum ContributorMergeOption {
+  SourceOnly,
+  TargetOnly,
+}
+
 export const MergeResultsWizardContributorsTab = () => {
-  const { control } = useFormContext<Registration>();
+  const { t } = useTranslation();
+  const { control, formState, setValue } = useFormContext<Registration>();
   const { sourceResult } = useContext(MergeResultsWizardContext);
   const sourceContributors = sourceResult.entityDescription?.contributors ?? [];
 
   const targetContributors = useWatch({ name: 'entityDescription.contributors', control }) ?? [];
+  const initialTargetContributors = (formState.defaultValues?.entityDescription?.contributors ?? []) as Contributor[];
 
   return (
     <>
+      <RadioGroup
+        sx={{ gridColumn: '1/-1', display: 'grid', gridTemplateColumns: 'subgrid' }}
```

Comment:
Første gang jeg har brukt `subgrid`, men der sparte meg for mye styr her! Det gjør at vi kan få `RadioGroup` til å spanne over alle 3 kolonner i griden, og at griden over kan brukes for `FormControlLabel` under der igjen. Uten dette måtte vi sikkert hacka oss frem til en løsning ved å lage enda en grid, som vi måtte passet på at samsvarer med griden som ligger over